code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -111,7 +112,6 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(/datum/gas/oxygen, /datum/g
 	gas_overlay = "nitryl"
 	moles_visible = MOLES_GAS_VISIBLE
 	dangerous = TRUE
-	fusion_power = 16
```

Comment:
All fusion power numbers will be changed/removed/added, they are used to calculate the instability of the reaction and I need to balance it. Ignore those for the time being since they are going to change a lot